File: My Graph/My Graph/AEdge.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace My_Graph
{
    class AEdge
    {
        private char cStart;
        private char cEnd;
        private int weight;

        public int getWeight()
        {
            return this.weight;
        }

        public char getStart()
        {
            return this.cStart;
        }

        public char getEnd()
        {
            return this.cEnd;
        }
        
        public AEdge(char cStart, char cEnd, int weight)
        {
            this.cStart = cStart;
            this.cEnd = cEnd;
            this.weight = weight;


        }



    }
}

File: Graph/GraphAdjList/AEdge.cpp
#include "stdafx.h"
#include "AEdge.h"

AEdge::AEdge(char start, char end, int weight)
{
	cStart = start;
	cEnd = end;
	iWeight = weight;
}
AEdge::~AEdge()
{
	
}
char AEdge::GetStart()
{
	return cStart;
}

char AEdge::GetEnd()
{
	return cEnd;
}